package.json

```diff
@@ -57,12 +57,13 @@
     "minimatch": "3.0.3",
     "mz": "2.6.0",
     "node-firefox-connect": "1.2.0",
-    "regenerator-runtime": "0.10.1",
     "parse-json": "2.2.0",
+    "regenerator-runtime": "0.10.0",
```

Comment:
I didn't touch package.json at all while resolving the conflicts. There was no conflict in this file that needed to be resolved manually. I get git did it on its own?